requirements.txt

```diff
@@ -1,4 +1,4 @@
-databricks-sql-connector~=2.7.0
+databricks-sql-connector>=2.7.0
```

Comment:
Any reason not to simply pin to the latest connector version? I'm okay with a wider specification, but it might be nicer to just hard-code to the latest and greatest.

Reply:
The only issue is keeping them in sync.  Per our discussion this morning, I'm ok updating to >= 2.9.3, <3.  If you make optional improvements in databricks-sql-connector, I don't want to force this project to have a release just to get access to them.